docs/kb/semgrep-ci/bitbucket-triggering-events-to-jenkins.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+  - Bitbucket
+  - Jenkins
+  - webhooks
+description: How to trigger events from the Bitbucket server to a Jenkins instance
+---
+# Triggering events from Bitbucket to Jenkins
+
+
+This guide walks you through connecting your Bitbucket source code manager (SCM) with Jenkins, a CI/CD environment, through the use of **webhooks**. By connecting Bitbucket with Jenkins, you are able to customize jobs, such as your Semgrep job, based on push or pull events from Bitbucket.
+
+## Defining a Bitbucket webhook 
+One way to connect Bitbucket (Cloud or Server) to a Jenkins instance is defining a Bitbucket webhook.
+It allows you to trigger builds on Bitbucket's push and pull requests events.
+
+:::info Prerequisites
+* You must install the [<i class="fas fa-external-link fa-xs"></i> Bitbucket Push and Pull Request](https://plugins.jenkins.io/bitbucket-push-and-pull-request/) plugin on your the Jenkins server. This method requires that your Jenkins instance is compatible with this plugin.
+* Jenkins and Bitbucket must be visible to each other. It means Bitbucket will invoke Jenkins after a push or pull request event and Jenkins will send the result/comments to Bitbucket.
```

Comment:
In this line, you wrote:
>... Bitbucket will invoke Jenkins after a push or pull ...

The wording implies that this is the effect that happens AFTER the user sets up the webhook. So it's not actually something the user has to do beforehand? If so, can we remove the bullet?

Reply:
Yes, we can remove the bullet.